R/ncaa_fb.R

```diff
@@ -14,17 +14,16 @@ SEASON_TYPES <- list(
 
 
 get_ncaa_season <- function(season, include_future = F) {
-  print(season)
   # TODO: dynamically get number of weeks
   regular <- seq_len(15) %>%
-    map_df(~.get_week(season, .x, SEASON_TYPES$regular, include_future))
-  bowls <- .get_week(season, 1, SEASON_TYPES$post, include_future) %>% 
+    map_df(~.get_ncaa_week(season, .x, SEASON_TYPES$regular, include_future))
```

Comment:
Bump for https://github.com/NathanDeMaria/EndGame/issues/5 ...had to do this b/c it's a package, so the later loaded script's `.get_week` would get used for both. Ew